### DISCUSSION TITLE
Url rewrite with different locale domain

### DISCUSSION THREAD

**Discussion by fenes** [vercel/next.js#36235] [Help]
I want to rewrite some URLs for each locale with a different domain.
Here is i18N config;
```
i18n: {
    localePath: path.resolve("./public/locales"),
    defaultLocale: 'tr',
    locales: ['en', 'tr'],
    localeDetection: false,
    domains: [
      {
        domain: 'www.domain.com',
        defaultLocale: 'en',
      },
      {
        domain: 'www.domain.com.tr',
        defaultLocale: 'tr',
        http: true
      },
    ],
    rewrites: [
      {
        destination: '/tr/hakkimizda',
        source: '/aboutus',
        locale: false,
      },
      {
        source: '/tr/contactus',
        destination: '/tr/iletisim',
        locale: false,
      }
    ],
  }
```
But I'm stuck into these rewrites block?
Any suggestions??
Thanks.

**Comment by fenes** [comment]
For example,
I want to rewrite this URL => domain.com.tr/hakkimizda 
here is the working url => domain.com.tr/aboutus
when I type domain.com.tr/hakkimizda , it has to be directed to aboutus(domain.com.tr/aboutus) page with TR locale

**Reply by bonesoul** [reply]
@fenes did you find a solution for this already?

**Reply by fenes** [reply]
unfortunately , no  :(

**Reply by bonesoul** [reply]
i've a similar issue here which i described in brief: https://github.com/vercel/next.js/discussions/38607
